### PR TITLE
feat(contrib/ec2): add support for custom volume sizes

### LINKED ISSUE #3682: contrib/ec2: Make volume size customizable
This should be a parameter users can override. Currently, we default to 100GB for the Docker volume and 10GB for the etcd volume.

### PR DESCRIPTION
**Pull request by carmstrong**
closes #3682

### CODE REVIEW

**Comment by carmstrong** [comment]
I manually tested both the default path and customizing all of these.

**Comment by kalbasit** [comment]
:+1:

File: docs/installing_deis/aws.rst [hunk 1]
```diff
@@ -102,11 +102,24 @@ in `cloudformation.json`_. For example, to configure all of the optional setting
     {
         "ParameterKey":     "ELBScheme",
         "ParameterValue":   "internal"
+    },
+    {
+        "ParameterKey":     "RootVolumeSize",
+        "ParameterValue":   "100"
```
Comment (bacongobbler) [inline docs/installing_deis/aws.rst]: When someone comes to the docs, they may be confused on what unit of measurement this is. Maybe we should add a note stating that this is in gigabytes?

**Comment by bacongobbler** [comment]
code LGTM

File: docs/installing_deis/aws.rst (hunk 1 above)
Comment (carmstrong) [inline docs/installing_deis/aws.rst]: They may also be confused as to what "internal" means - these options all have detailed descriptions in deis.template.json, so I'd rather not repeat those here. I tried to call them to the other file with "The default values are defined in deis.template.json."

Comment (bacongobbler) [inline docs/installing_deis/aws.rst]: Ah, I skipped over that. :+1:

**Comment by mboersma** [comment]
Code LGTM.

**Pull request merged**